department-of-veterans-affairs/vets-website
# Update pension introduction content

## Summary
On the Pension Benefits application(21P-527EZ),
On the Introduction page

Update content

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
Unauthenticated
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Verify application redirects you to the Introduction page
3. Verify content is updated to match the acceptance criteria

Authenticated 
1. Sign in via `http://localhost:3001/sign-in/mocked-auth` to authenticate your user
2. Go to `http://localhost:3001/pension/application/527EZ` in your browser
3. Verify application redirects you to the Introduction page
4. Verify content is updated to match the acceptance criteria

Saved
1. Sign in via `http://localhost:3001/sign-in/mocked-auth` to authenticate your user
2. Go to `http://localhost:3001/pension/application/527EZ` in your browser
3. Verify application redirects you to the Introduction page
4. Start new application
5. Select 'Finish this application later' on any page
6. Verify you are taken to the application saved page

Continue
1. Sign out with a saved application
2. Sign in via `http://localhost:3001/sign-in/mocked-auth` to authenticate your user
3. Go to `http://localhost:3001/pension/application/527EZ` in your browser
4. Verify application redirects you to the Introduction page
5. Verify that the resume application UI is showing

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69745)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots
_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

### Desktop
| Before | After | Authenticated | Saved | Continue |
| ------ | ------ | ------ | ------ | ------ |
|![staging va gov_pension_application_527EZ_introduction](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/2866b796-8d2d-4c73-b796-6b91ad353b60)|![localhost_3001_pension_application_527EZ_introduction](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/dcb428b1-1a9e-42af-8a67-04cdd20fda0f)|![localhost_3001_pension_application_527EZ_ (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/743c5c9a-8362-48a7-a939-5062257b43ef)|![localhost_3001_pension_application_527EZ_ (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/fd5b7e2e-acbe-4a00-aa7e-06aae5d214fd)|![localhost_3001_pension_application_527EZ_](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/0c106a95-ce45-4a28-a3cf-7848cbf7a7fa)

## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
The intro page section of the 527EZ (Apply for Veterans Pension page), will have updated copy and and links within the Prepare section, as shown on the designs.

### Quality Assurance & Testing

### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
